Formula/qt5.rb

```diff
@@ -27,28 +26,6 @@ class Qt5 < Formula
     sha256 "73d33dd2563c39542844c276a7bd43463f2974fde141e7afeb3057168adbe606" => :mavericks
   end
 
-  # Restore `.pc` files for framework-based build of Qt 5 on OS X. This
```

Comment:
I suspect it's still going to be very breaking, especially as Homebrew moves stuff from `qt` to `qt5`. The Qt ecosystem isn't exactly known for its rapid response to upstream changes. Up to y'all though, I'm no longer around really, just noticed this and thought I'd add the comment.